sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Is there a default or recommend file format? Or anything is fine as long as you customize the payload and recommender?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
No documentation for it... I just used CSV file.